lte/gateway/c/oai/tasks/mme_app/mme_app_ip_imsi.h

```diff
@@ -20,7 +20,6 @@ extern "C" {
 #include "log.h"
 #include "common_defs.h"
 #include "common_types.h"
-void initialize_ipv4_map(void);
 int mme_app_insert_ue_ipv4_addr(uint32_t ipv4_addr, imsi64_t imsi64);
```

Comment:
please add documentation to each of these public functions

Reply:
Added description in file, mme_app_ip_imsi.cpp
https://github.com/magma/magma/pull/7125/files#diff-1b5fb3ec21f41be0470796c1df743a332a0af3dd3f77d85a324a3303efca6acbR44
Do we need provide description in both files, mme_app_ip_imsi.h and mme_app_ip_imsi.cpp?

Reply:
missed the docs on the other one, thanks!